app/src/main/java/com/ciandt/dragonfly/example/features/projectselection/ProjectSelectionActivity.kt

```diff
@@ -59,7 +63,16 @@ class ProjectSelectionActivity : BaseActivity(), ProjectSelectionContract.View {
         super.onResume()
         presenter.attachView(this)
 
-        if (projects.isEmpty()) {
+        if (firstTime) {
+            firstTime = false
+
+            showLoading()
+
+            Handler().postDelayed({
+                presenter.loadProjects()
```

Comment:
User opens the app, and loadProjects from database.
At the same time, Firebase starts doing a sync.

As local database query is faster than firebase sync, user always get the old result.
Also, if it's the user's first time, the database will be empty and a empty-state will be show.

So, in these conditions, I added a delay when user open the screen for the first time.
ProjectSelectionActivity is created with `firstTime = true` and at first onResume(), I check and wait for some seconds to wait for Firebase to sync.

If user opens RealTimeActitivity and go back, ProjectSelectionActivity will have `firstTime = false`, because in 99% of time, this activity wont be destroyed and the loading wont be delayed.